From 1ebc89120c23a965131211b566f7058ec92f2cc9 Mon Sep 17 00:00:00 2001
From: kitakar5525 <34676735+kitakar5525@users.noreply.github.com>
Date: Thu, 21 Nov 2019 10:56:45 +0900
Subject: [PATCH 2/4] mwifiex: s0ix test: add __maybe_unused to
 mwifiex_pcie_suspend/mwifiex_pcie_resume

Add __maybe_unused to old mwifiex_pcie_suspend/mwifiex_pcie_resume
functions to suppress warning on build.

---
 drivers/net/wireless/marvell/mwifiex/pcie.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/marvell/mwifiex/pcie.c b/drivers/net/wireless/marvell/mwifiex/pcie.c
index c07dfd73b..971fe4c2b 100644
--- a/drivers/net/wireless/marvell/mwifiex/pcie.c
+++ b/drivers/net/wireless/marvell/mwifiex/pcie.c
@@ -147,7 +147,7 @@ static bool mwifiex_pcie_ok_to_access_hw(struct mwifiex_adapter *adapter)
  * If already not suspended, this function allocates and sends a host
  * sleep activate request to the firmware and turns off the traffic.
  */
-static int mwifiex_pcie_suspend(struct device *dev)
+static int __maybe_unused mwifiex_pcie_suspend(struct device *dev)
 {
 	struct mwifiex_adapter *adapter;
 	struct pcie_service_card *card;
@@ -192,7 +192,7 @@ static int mwifiex_pcie_suspend(struct device *dev)
  * If already not resumed, this function turns on the traffic and
  * sends a host sleep cancel request to the firmware.
  */
-static int mwifiex_pcie_resume(struct device *dev)
+static int __maybe_unused mwifiex_pcie_resume(struct device *dev)
 {
 	struct mwifiex_adapter *adapter;
 	struct pcie_service_card *card;
-- 
2.24.0

